lein-cljfmt/src/leiningen/cljfmt.clj

```diff
@@ -8,8 +8,12 @@
             [leiningen.cljfmt.diff :as diff]
             [meta-merge.core :refer [meta-merge]]))
 
+(defn remove-dir-from-filename [dir filename]
+  (str/replace filename dir ""))
```

Comment:
Good call. Especially bad if someone has their project in a root directory, like:

```bash
c/myproject/src/org/...
```

well, now every "c" in the filenames will be removed.

Your version fixes that, I'll make the switch.